petastorm/unischema.py

```diff
@@ -132,8 +133,12 @@ def create_schema_view(self, fields):
         """
 
         # Split fields parameter to regex pattern strings and UnischemaField objects
-        regex_patterns = [f for f in fields if isinstance(f, str)]
-        unischema_field_objects = [f for f in fields if not isinstance(f, str)]
+        regex_patterns = [f for f in fields if isinstance(f, string_types)]
+        unischema_field_objects = [f for f in fields if isinstance(f, UnischemaField)]
```

Comment:
Per the title, I assume that beforehand we were only allowing for `str`s, but now `string_types` should include Unicode strings?

Reply:
Yep.